programs/common.c

```diff
@@ -316,6 +316,8 @@ static OUTPUT_FORMAT_MAP format_map [] =
 	{	"prc",		0,	SF_FORMAT_WVE	},
 	{	"ogg",		0,	SF_FORMAT_OGG	},
 	{	"oga",		0,	SF_FORMAT_OGG	},
+	{	"opus",		0,	SF_FORMAT_OGG	},
+	{	"opus",		0,	SF_FORMAT_OPUS	},
```

Comment:
I don't think this could ever be hit, as the `"opus"` above would mask it.

Also, not sure how to accomplish this, but mapping "opus" to SF_FORMAT_OGG or SF_FORMAT_OPUS alone seems wrong. Could it be possible to change this to map "opus" to SF_FORMAT_OGG | SF_FORMAT_OPUS, but then one is mapping a subtype as well.

`sfe_file_type_of_ext` could be modified to detect if format_map[].format has set a subtype, and if so ignore the format passed as an argument. Thoughts?

Reply:
Yeah, you're right. That was some last second futzing before I put the computer down for the evening.